.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "python/pybind11"]
 	path = python/pybind11
 	url = https://github.com/pybind/pybind11.git
+[submodule "ext/fmt"]
```

Comment:
please make sure that fmt can be taken from the environment, e.g., spack (https://spack.readthedocs.io/en/latest/package_list.html#fmt) and specify the required version

Reply:
Should be working now.